dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/cache/Properties.java

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.websphere.simplicity.config.cache;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlType;
+
+/**
+ * Configuration for the following nested elements:
+ *
+ * <ul>
+ * <li>cacheManager --> properties</li>
+ * </ul>
+ */
+@XmlType(name = "CacheManagerProperties")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class Properties {
+    @XmlAttribute(name = "infinispan.client.hotrod.auth_realm")
```

Comment:
These are infinispan specific. The Properties element is used all over (not just for JCache, sessionCache uses it, as do others).

Instead you should use @XmlAnyAttribute and see if you can make that work. I think you annotate a map and it will allow you to assign any attribute name any value.